pages/news.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <v-container>
-    <v-row>
+    <v-row> 
+      <v-sheet v-if="notLoaded" class="mx-3">
+        <h1 class="mt-3">
+          {{ $t('news.title') }}
+        </h1>
+        <p class="mb-6">
+          {{ $t('news.blurb') }}
+        </p>
```

Comment:
This can be moved outside the `<v-row>` (below `<v-container>`) and without the extra classes. And likewise in the `<v-col v-if="loaded">` section, you can remove the duplicate `news.title` and `news.blurb` too.
- it looks like other pages have this header and paragraph element within a `v-row` and `<v-col>` too that I overlooked. If you don't mind, can you edit all those pages too and move them (under the `<v-container>`).

```suggestion
<h1>{{ $t('news.title') }}</h1>
<p>{{ $t('news.blurb') }}</p>
<v-sheet v-if="!loaded">
   <v-skeleton-loader type="card-avatar, article" elevation="2" class="my-5" height="400" />
   <v-skeleton-loader type="card-avatar, article" elevation="2" class="my-5" height="400" />
   <v-skeleton-loader type="card-avatar, article" elevation="2" class="my-5" height="400" />
</v-sheet>
```